<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtApplicationClass</class>
 <widget class="QMainWindow" name="QtApplicationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网络共享器</string>
  </property>
  <property name="toolTipDuration">
   <number>-5</number>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>366</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_speed">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_mac_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>621</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>700</width>
      <height>311</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <column>
     <property name="text">
      <string>协议</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>IP:端口</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>发送</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>接收</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>机器码</string>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>718</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menus">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionstop"/>
    <addaction name="separator"/>
    <addaction name="actionset"/>
   </widget>
   <addaction name="menus"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_start">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>-</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>启动</string>
   </property>
  </action>
  <action name="actionstop">
   <property name="text">
    <string>停止</string>
   </property>
  </action>
  <action name="actionset">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtApplication.qrc"/>
 </resources>
 <connections/>
</ui>
